#display debug information
print_debug: false

#Goal offset (meters), should be set to 0
goal_offset_x: 0.0 
goal_offset_y: 0.0

#passthrough filter (z and x are the 
#height and distance normal to camera plane respectively)
min_x: 0.0 #minimum width
max_x: 1.1 #maximum width
min_z: 0.18 #minimum height
max_z: 0.4 #maximum height

#voxel parameters
leaf_size: 0.01 #in meters
min_points_voxel: 1 #minimum number of point clouds to constitute a voxel

#Statistical Outlier Removal
sor_knn: 30 #k nearest neigbours to consider
sor_stddev: 1.0 #standard deviation

#Radius Outlier removal
# outrem_rad_search: 0.04 
# outrem_min_neigh: 10

#Planar segmentation
seg_max_itr: 100      #number of iterations to refine
seg_dist_thresh: 0.01 #maximum distance between points on the plane

#euclidean cluster extraction
ec_tol: 0.02          #if too high, multiple objects seen as one cluster
ec_min_size: 50       #minimum no. of points required to constitute a cluster
ec_max_size: 2500     #maximum no. of points required to constitute a cluster